subset/bacnet/bacnetTests/src/main/java/FauxDeviceEngine/EntryPoint.java

```diff
@@ -0,0 +1,113 @@
+package FauxDeviceEngine;
+
+import com.serotonin.bacnet4j.LocalDevice;
+import com.serotonin.bacnet4j.npdu.ip.IpNetwork;
+import com.serotonin.bacnet4j.obj.BACnetObject;
+import com.serotonin.bacnet4j.transport.Transport;
+import com.serotonin.bacnet4j.type.enumerated.ObjectType;
+import com.serotonin.bacnet4j.type.enumerated.PropertyIdentifier;
+import com.serotonin.bacnet4j.type.primitive.CharacterString;
+import helper.FileManager;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class EntryPoint {
+
+    private static final int deviceId = (int) Math.floor(Math.random() * 1000.0);
```

Comment:
nice hack, deviceId should come from JSON

Reply:
I updated the code to read the device id from JSON and if not provided it will generate a random one.